src/ContextMenuTrigger.js

```diff
@@ -49,26 +53,37 @@ export default class ContextMenuTrigger extends Component {
         }
     }
 
+
     handleTouchEnd = (event) => {
         event.preventDefault();
         clearTimeout(this.touchstartTimeoutId)
     }
 
+    handleMouseMove = (event)=>{
+        event.persist()
+        this.mouseMoveEvent = event
```

Comment:
You do not need to doubt, it certainly worked, in fact, I don‘t want to cache the coordinate values, I hope to get the real-time coordinates.